bindings/python/mata.pyx

```diff
@@ -139,18 +139,38 @@ cdef class Nfa:
     cdef mata.CNfa *thisptr
     cdef alphabet
 
-    def __cinit__(self, state_number = 0, alphabet=None):
+    def __cinit__(self, state_number = 0, alphabet = None):
         """Constructor of the NFA
 
         :param int state_number: number of states in automaton
-        :param Alpabet alphabet: alphabet corresponding to the automaton
+        :param Alphabet alphabet: alphabet corresponding to the automaton
         """
         self.thisptr = new mata.CNfa(state_number)
         self.alphabet = alphabet
 
     def __dealloc__(self):
         del self.thisptr
 
+    @property
+    def initialstates(self):
```

Comment:
Please use snake case (initial_states and final_states).

Btw. I'm wondering if this is necessary (I think, both are private in Nfa, hence there were no setters, and there are only public functions for manipulating with these properties).

Reply:
This _lowercase_ case is unfortunately used in libMata. I used the same naming convention here to keep it consistent (even though I would rather rename the libMata Nfa member variables to _snake_case_, too).

And, due to us using `struct` instead of `class` for `Mata::Nfa::Nfa`, all member variables are public by default. Furthermore, the library is built with the intent for the user to access the member variables directly. For the consistency with libMata, I would say the setters and getters are appropriate. And there are only some public functions to manipulate with them. Without a direct access, you could not iterate over initial/final states, for example.

Reply:
Yes, I'm aware of that, hence, you can "simulate" the privateness, by not making it property :P

However, I have no issues with keeping it public, though I would prefer the snake case (you may have noticed that I actually changed some names in the python API). We can discuss this on Thursday. Consider this resolved then.

Reply:
That could be done. I will use _snake_case_ here.

Reply:
Make a mental (or some other) note, that we should discuss this on Thursday. There is actually a different reasoning for the sorting and I see, why this is problem for the product (naturally, the order matters here), however, for subsets I don't think the order matters (those are macrostates, which are sets; the sorting is performed so the resulting tuple is deterministic, i.e. the same everytime the code is run; the issue is that sets cannot be hashed, hence we convert and use the tuples instead).